projectfiles/CodeBlocks/libraries.md

```diff
@@ -24,7 +24,7 @@ The other libraries require complicated compilation procedures too in-depth to d
 ## Updating Boost libraries
 
 Download and unpack the source of the libraries zlib, libbzip2, boost (version 1.64 or 1.68 preferred)
-http://www.bzip.org/downloads.html  ; http://www.boost.org/users/download/ ; http://www.zlib.net/
+http://www.bzip.org/downloads.html  ; http://www.boost.org/users/download/ ; http://www.zlib.net/.
```

Comment:
Adding a period here is probably a mistake, it might confuse some viewers. This looks like it should be presented as a list, with each URL on a separate line.